src/graphorm/index.ts

```diff
@@ -490,6 +492,26 @@ const obj = new GraphORM({
       title: createSmartTitleField({
         field: 'title',
       }),
+      translation: {
+        jsonType: true,
+        transform: (
+          translations: Partial<Record<ContentLanguage, PostTranslation>>,
+          ctx: Context,
+        ): Partial<Record<keyof PostTranslation, boolean>> => {
+          if (!ctx.contentLanguage || !translations[ctx.contentLanguage]) {
+            return {};
+          }
+
+          const translation = translations[ctx.contentLanguage]!;
```

Comment:
Better to avoid `!`, typescript will automatically know `translation` is defined after early return:
```suggestion
          const translation = ctx.contentLanguage
            ? translations[ctx.contentLanguage]
            : undefined;

          if (!translation) {
            return {};
          }

```